app/src/main/java/ca/bc/gov/bchealth/ui/healthrecord/add/AddHealthRecordsFragment.kt

```diff
@@ -54,15 +69,26 @@ class AddHealthRecordsFragment : Fragment(R.layout.fragment_health_records) {
                 }
             )
 
+        savedStateHandle.getLiveData<Boolean>(
```

Comment:
change as per reference of HealthPassFragment

Reply:
done